src/context/LocationsContext.js

```diff
@@ -10,6 +10,7 @@ const LocationsContextProvider = (props) => {
   //state is updated in the Map component
   const [newCenter, setNewCenter] = useState();
   const [loading, setLoading] = useState({ loading: false, message: '' });
+  const [id, setId] = useState();
```

Comment:
`id` is a pretty generic variable name. Could we rename this to `watchId` perhaps?